core/src/org/pentaho/di/core/database/DatabaseInterface.java

```diff
@@ -1152,9 +1152,17 @@ public String getModifyColumnStatement( String tablename, ValueMetaInterface v,
    * Customizes the ValueMetaInterface defined in the base
    *
    * @param v the determined valueMetaInterface
-   * @param valueType the sql column type
+   * @param rm the sql column type
    * @return ValueMetaInterface customized with the data base specific types
```

Comment:
Since you were in here cleaning up the javadoc for this method, it would have been a good idea to complete the clean and add the index field (3rd parameter) that is missing as well...